### PR TITLE
Add Workflow Commons module to the Marketplace Guide

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

File: content/appstore/modules/workflow-commons.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    2. User role does the following:
+        1. Executes workflows by completing user tasks.
+        2. Views personal performance in MyTaskDashboard.
+
+## 4 Usage
+
+### 4.1 Page Templates
+
+#### 4.1.1 My Initiated Workflows
+Allows users who initiate workflows to keep track of their progress. A user is also able to `Withdraw` a workflow if it is still `In progress`.
```
Comment (MariaShaposhnikova) [inline content/appstore/modules/workflow-commons.md:79]: Is **Withdraw** a button?

File: content/appstore/modules/workflow-commons.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+#### 4.1.2 Workflow Task Dashboard
+Shows users who execute tasks, for example, a task-based dashboard.
```
Comment (MariaShaposhnikova) [inline content/appstore/modules/workflow-commons.md:82]: Is this page template only for users who _execute_ task, can it be an admin who just _monitors_ them?

File: content/appstore/modules/workflow-commons.md (hunk 1 above)
Comment (MitchelMx) [reply content/appstore/modules/workflow-commons.md:79]: Withdraw is indeed a button, but you could also consider it an action to be executed on an `In progress` workflow.

File: content/appstore/modules/workflow-commons.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+The [Workflow Commons](https://marketplace.mendix.com/link/component/117066) module provides an out-of-the box starter experience for users who would like to build a workflow in their app.
+
+### 1.1 Typical Use Cases
+
+This module allows Mendix developers with little or no experience in building workflows to start with standardized options for implementing and monitoring workflows.
+
+### 1.2 Features
+
+* Dashboards for monitoring workflows, for administrators, initiators, and executors of workflows
```
[the remark this answers is not in the document]
Comment (MitchelMx) [reply content/appstore/modules/workflow-commons.md:18]: Executors are any users that have the ability to chose an outcome/complete a user task.

File: content/appstore/modules/workflow-commons.md (hunk 2 above)
Comment (MitchelMx) [reply content/appstore/modules/workflow-commons.md:82]: Admins have the Workflow Dashboard, Workflow Task Dashboard is specifically for executing users.

File: content/appstore/modules/workflow-commons.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+3. Open the **MendixSSOUserVariant** folder.
+   4. Select all pages and the microflow in this folder (select multiple items by selecting the first item and then keep <kbd>Ctrl</kbd> pressed to select other items).
+   2. Right-click and choose **Exclude from project**.
+4. Open the **SystemUserVariant** folder.
+   7. Select all pages and the microflow in this folder.
+   2. Right-click and choose **Include in project**.
+
+## 3 Configuration
+
+1. Add the following pages to your navigation:
```
[the remark this answers is not in the document]
Comment (MitchelMx) [reply content/appstore/modules/workflow-commons.md:61]: Yes these are all pages that the module provides ready to use.

File: content/appstore/modules/workflow-commons.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (MariaShaposhnikova) [reply content/appstore/modules/workflow-commons.md:18]: I will add this explanation to the document.

File: content/appstore/modules/workflow-commons.md (hunk 2 above)
Comment (MariaShaposhnikova) [reply content/appstore/modules/workflow-commons.md:82]: Thanks, this helps to choose styling, I don't think we need code styling here in this case.

File: content/appstore/modules/workflow-commons.md (hunk 1 above)
Comment (MariaShaposhnikova) [reply content/appstore/modules/workflow-commons.md:79]: Thanks, this helps to choose styling, I don't think we need code styling here in this case.

File: content/appstore/modules/workflow-commons.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
+---
+title: "Workflow Commons"
+category: "Modules"
+description: "Describes usage and implementation the Workflow Commons module that is used with workflows."
```
Comment (PaulJongenMx) [inline content/appstore/modules/workflow-commons.md:4]: "implementation the" I think this does not run correctly. Expect something like: "implementation of the"

File: content/appstore/modules/workflow-commons.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+### 1.2 Features
+
+* Dashboards for monitoring workflows for administrators, initiators, and executors (users who can chose an outcome/complete a user task) of workflows
+* Page templates to generate task pages, task admin, an inbox, and different dashboards
+* Attachments layer on top of workflows
+* Comments section to use with workflows
+
+### 1.3 Limitations
+
+* Requires [Mendix SSO](https://marketplace.mendix.com/link/component/117212) to fully function directly after the installation. If you do not want to use Mendix SSO, you will have to copy contents to your own module and use templates to build pages.
```
Comment (PaulJongenMx) [inline content/appstore/modules/workflow-commons.md:25]: This section can be removed (because it's explained in 2.1 how to use WFC without MxSSO).

Comment (MariaShaposhnikova) [reply content/appstore/modules/workflow-commons.md:25]: We have this section in our template, but I think we can skip this indeed, as it is explained in Dependencies.

**Pull request merged**